pkg/status/render.go

```diff
@@ -158,16 +207,32 @@ func FormatSecurityAgentStatus(data []byte) (string, error) {
 	var b = new(bytes.Buffer)
 
 	stats := make(map[string]interface{})
-	json.Unmarshal(data, &stats) //nolint:errcheck
+	if err := json.Unmarshal(data, &stats); err != nil {
+		if err := renderErrors(b, []error{err}); err != nil {
+			fmt.Println(err)
+			return "", nil
+		}
+		return b.String(), nil
+	}
```

Comment:
This block of code appears several times in this file. Can you create a function that performs these actions?